recommended.code-workspace

```diff
@@ -64,6 +61,7 @@
       "ms-vscode.vscode-typescript-tslint-plugin",
       "redhat.vscode-yaml",
       "silvenon.mdx",
+      "jounqin.vscode-mdx",
```

Comment:
when working with imports it turned out that using the "normal" MDX editor (instead of forcing the markdown editor) with this one which esLints the MDX is more helpful.  Let's try whether it's overall better or not.   At least for the kit part I liked having more code-ish editors